src/core/lib/surface/server.cc

```diff
@@ -633,7 +633,7 @@ static void start_new_rpc(grpc_call_element* elem) {
     for (i = 0; i <= chand->registered_method_max_probes; i++) {
       rm = &chand->registered_methods[(hash + i) %
                                       chand->registered_method_slots];
-      if (!rm) break;
+      if (!rm->server_registered_method) break;
```

Comment:
Nit: I'd prefer to see this written as `!= nullptr`, to make it clear to the reader that it's a pointer and not a bool.

Reply:
Done. Thanks for the effort to improve the quality of our codebase :)